torch/nn/parallel/distributed.py

```diff
@@ -633,6 +633,9 @@ def _register_comm_hook(self, state: object, hook: callable):
                              c10d reducer would call this hook and use the tensors returned
                              by the Future and copy grads to individual parameters.
 
+                             We also provide an API called "get_future" to retrieve a future
```

Comment:
Add a warning to highlight that this future is different from `torch.futures.Future` and explain the differences.

Reply:
use `` instead of " to wrap code.

Reply:
@mrshenli Could you elaborate more on the differences between `torch.futures.Future` and `torch._C.Future`? The former is used in jit and latter is just the `at::ivalue`'s Future?

Reply:
`torch.futures.Future` and `torch._C.Future` is roughly the same, where the former is sheer wrap of the latter. The concern is actually that the return future type of `get_future` does not behave the same way as `torch.futures.Future`, so that we need to highlight that.